### PR TITLE
UI Patch 1 

### PR DESCRIPTION
**Pull request by TeeKunGZ**

### CODE REVIEW

File: UI_Thai.tsv [hunk 1]
```diff
@@ -1,148 +1,140 @@
...
-UI_20150317_000047
-UI_20150317_000048
-UI_20150317_000049
-UI_20150317_000050
-UI_20150317_000051
-UI_20150317_000052
-UI_20150317_000053
-UI_20150317_000054
-UI_20150317_000055
+An unexpected event is available.{/}
```
Comment (Zixties60) [inline UI_Thai.tsv:1]: Line 1-9 is missing.

File: UI_Thai.tsv [hunk 2]
```diff
@@ -1,148 +1,140 @@
...
+UI_20150317_000039	{@st49}ลบ
+UI_20150317_000040	{@st44}บันทึก{/}
+UI_20150317_000041	{@st44}ยกเลิก{/}
+UI_20150317_000042	{@st43}{s22}ร้านค้าทรงผม{/}
+UI_20150317_000043	{@st59}ทรงผมก่อนหน้า{/}
+UI_20150317_000044	{@st59}ทรงผมถัดไป{/}
+UI_20150317_000045	{@st41b} เปลี่ยน {s18} (ต้องการ 100 iCoins) {/}}
+UI_20150317_000046	{@st59}เปลี่ยนทรงผมเรียบร้อยแล้ว{/}
+UI_20150317_000047	{@st41}ยกเลิก
+UI_20150317_000048	{@st43}คลาดมืด{/}
```
Comment (Zixties60) [inline UI_Thai.tsv:40]: Miss spell, Please change to "ตลาดมืด"

File: UI_Thai.tsv [hunk 3]
```diff
@@ -1,148 +1,140 @@
...
+UI_20150317_000012	{@st41b}ครั้งเดียว{/}
+UI_20150317_000013	{@st41b}ทั้งหมด{/}
+UI_20150317_000014	{@st41b}หยุดชั่วคราว{/}
+UI_20150317_000015	{@st43}บันทึกการผจญภัย{/}
+UI_20150317_000016	{@st43}ตลาดประมูลไอเท็ม{/}
+UI_20150317_000017	{@st59}ปิดตลาดประมูลไอเท็ม{/}
+UI_20150317_000018	{memo X}{@st41b}ไอเท็มที่ขายได้ : ×%s, ไอเท็มที่ยังไม่ได้ขาย : ×%s{/}
+UI_20150317_000019	{@st43}{s22}ประมูลไอเท็ม{/}
+UI_20150317_000020	{@st59}ปิดข้อมูลเควส{/}
+UI_20150317_000021	Availest
```
Comment (Zixties60) [inline UI_Thai.tsv:13]: This line wasn't translated to Thai.

File: UI_Thai.tsv [hunk 4]
```diff
@@ -1,148 +1,140 @@
...
+UI_20150317_000022	{@st41b}เวลาที่เหลือ
+UI_20150317_000023	3 ชั่วโมง16 นาที
+UI_20150317_000024	{@st41b}ราคาเริ่มต้น
+UI_20150317_000025	{@st41b}Bids
```
Comment (Zixties60) [inline UI_Thai.tsv:17]: This line wasn't translated to Thai.

File: UI_Thai.tsv [hunk 5]
```diff
@@ -1,148 +1,140 @@
...
+UI_20150317_000026	{@st41b}ราคาปัจจุบัน
+UI_20150317_000027	{@st41b}ราคาประมูล
+UI_20150317_000028	{@st41b}ราคาประมูลขั้นต่ำเริ่มต้นที่ {@st57}%s{@st41b}silver{/}
+UI_20150317_000029	{@st41b}ประมูล{/}
+UI_20150317_000030	{@st43b}ซื้อเลย !{nl}{img icon_item_silver 20 20}{@st57}%s{/}
+UI_20150317_000031	{@st44}Permission Settings{/}
+UI_20150317_000032	{@st45}ชื่อ : %s{/}
+UI_20150317_000033	{@st45}คลังเก็บของปาร์ตี้{/}
+UI_20150317_000034	{@st49}None
```
Comment (Zixties60) [inline UI_Thai.tsv:26]: This line wasn't translated to Thai.

File: UI_Thai.tsv [hunk 6]
```diff
@@ -1,148 +1,140 @@
...
+UI_20150317_000022	{@st41b}เวลาที่เหลือ
+UI_20150317_000023	3 ชั่วโมง16 นาที
+UI_20150317_000024	{@st41b}ราคาเริ่มต้น
+UI_20150317_000025	{@st41b}Bids
+UI_20150317_000026	{@st41b}ราคาปัจจุบัน
+UI_20150317_000027	{@st41b}ราคาประมูล
+UI_20150317_000028	{@st41b}ราคาประมูลขั้นต่ำเริ่มต้นที่ {@st57}%s{@st41b}silver{/}
+UI_20150317_000029	{@st41b}ประมูล{/}
+UI_20150317_000030	{@st43b}ซื้อเลย !{nl}{img icon_item_silver 20 20}{@st57}%s{/}
+UI_20150317_000031	{@st44}Permission Settings{/}
```
Comment (Zixties60) [inline UI_Thai.tsv:23]: This line wasn't translated to Thai.

File: UI_Thai.tsv [hunk 7]
```diff
@@ -1,148 +1,140 @@
...
-UI_20150317_000100
-UI_20150317_000101
-UI_20150317_000102
-UI_20150317_000103
-UI_20150317_000104
+UI_20150317_000083	{memo X}{@st41b}โจมตี {@sti7}%s{/}
+UI_20150317_000084	{memo X}{@st41b}สนับสนุน {@sti7}%s{/}
+UI_20150317_000085	{memo X}{@st41b}ความยาก{@sti7}%s{/}
+UI_20150317_000086	{@st59}ปิดหน้าต่างรายละเอียด Class{/}
+UI_20150317_000087	แจ้งเตื่อนการเปลี่ยน Class
```
Comment (Zixties60) [inline UI_Thai.tsv:79]: Miss spell "เตือน".

File: UI_Thai.tsv [hunk 8]
```diff
@@ -1,148 +1,140 @@
...
+UI_20150317_000088	{@st43} เปลี่ยนอาชีพ
+UI_20150317_000089	{@sti7}{s14} ตัวละคร {/}
+UI_20150317_000090	{@sti8}{s14} อาชีพ {/}
+UI_20150317_000091	{@st44}Chat{/}
```
Comment (Zixties60) [inline UI_Thai.tsv]: This line wasn't translated to Thai.

**Comment by imcgames** [comment]
Please leave a comment like "Good to merge" for us to understand it's correct PR or not. Thank you.

**Comment by imcgames** [comment]
Updated through a manual way. :+1:

**Pull request closed without merging**